module/applications/item/item-sheet.mjs

```diff
@@ -71,6 +71,7 @@ export default class ItemSheet5e extends ItemSheet {
       labels: item.labels,
       isEmbedded: item.isEmbedded,
       advancementEditable: (this.advancementConfigurationMode || !item.isEmbedded) && context.editable,
+      rollData: this.item.getRollData.bind(this.item),
```

Comment:
I think I'm missing an insight on why this helps us instead of just `rollData: this.item.getRollData()`. Is the idea that by passing a bound function it avoids potentially costly roll data preparation in cases where there are no inline rolls that require enrichment? I think I've probably just answered my own question, but I'll ask it anyways.